src/python/pants/backend/project_info/tasks/idea_plugin_gen.py

```diff
@@ -27,7 +27,7 @@
 _TEMPLATE_BASEDIR = 'templates/idea'
 
 # Follow `export.py` for versioning strategy.
-IDEA_PLUGIN_VERSION = '0.0.2'
+IDEA_PLUGIN_VERSION = '1.9.2'
```

Comment:
Any reason for 1.9.2? I was thinking `0.0.3` because https://github.com/pantsbuild/pants/blob/819f9a6de376a8d1daf58d53e7a073b3774b4d68/src/python/pants/backend/project_info/tasks/export.py#L60-L61

Reply:
Hm, despite the comment, I was still thinking that this meant the intellij pants plugin. The next release of that is `1.9.2`, ergo...

Will fix.